### PR TITLE
GEOPY-1147: Increment property group for data with new depth sampling

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1147 - Increment property group for data with new depth sampling**
Push through GEOPY-1144 first

### CODE REVIEW

File: las_geoh5/.idea/.gitignore
Comment (domfournier) [inline las_geoh5/.idea/.gitignore:1]: Can you add .ideas to the list of ignores?

**Review by domfournier — APPROVED** [review]
Looks good.

**Review by domfournier — APPROVED** [review]
Go for tests

**Review by domfournier — DISMISSED** [review]
Looks good. Let's finish the geoh5py PR, and update the lock file before merging.

File: pyproject.toml [hunk 1]
```diff
@@ -62,7 +62,6 @@ plugins = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--cov las_geoh5 --cov-report html --cov-report term-missing:skip-covered"
```
Comment (domfournier) [inline pyproject.toml:65]: Can you bring this back, and remove the `fail_under = 80` below? 
I can show you how to make it pass with pycharm.

**Review by domfournier — COMMENTED** [review]
Almost there. See comment.

**Review by domfournier — APPROVED** [review]
Great. Thanks for the changes.

File: .idea/scopes/sources.xml
Comment (sebhmg) [inline .idea/scopes/sources.xml:1]: restore file: copy the one from python-template repo, and rename my_app to las-geoh5

File: .idea/copyright/profiles_settings.xml
Comment (sebhmg) [inline .idea/copyright/profiles_settings.xml:1]: keep this file

File: .idea/copyright/MiraGeoscience.xml
Comment (sebhmg) [inline .idea/copyright/MiraGeoscience.xml:1]: restore file: copy the one from python-template repo, and rename `my_app` to `las-geoh5`

**Review by sebhmg — CHANGES_REQUESTED** [review]
retore the .idea files

**Review by domfournier — APPROVED** [review]
Looking good. Moving to test

File: .idea/scopes/sources.xml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <component name="DependencyValidationManager">
-  <scope name="sources" pattern="file:las_geoh5/*||file:tests/*" />
+  <scope name="sources" pattern="(file:las-geoh5//*||file:devtools/*||file:tests//*||file:README*.rst)&amp;&amp;!file:*//htmlcov//*&amp;&amp;!file:*//.coverage&amp;&amp;!file:*//.*//*" />
```
Comment (sebhmg) [inline .idea/scopes/sources.xml:2]:
```suggestion
  <scope name="sources" pattern="(file:las_geoh5//*||file:devtools/*||file:tests//*||file:README*.rst)&amp;&amp;!file:*//htmlcov//*&amp;&amp;!file:*//.coverage&amp;&amp;!file:*//.*//*" />
```

**Review by domfournier — APPROVED** [review]
Looks good. Just need to figure out what's wrong with poetry with @sebhmg

**Review by sebhmg — APPROVED** [review]

**Pull request merged**